### ISSUE TITLE
Add props to render arrays 

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#2441]
Of the ~60 files in /views/.map we need to: 

- [ ] Got through these ~60 files and Identify which ones use `map` for rendering. 
- [ ] Add requisite props to them to work with React .

[State of React](https://www.notion.so/hicommonwealth/12-30-State-of-React-40a0aa870e524dd78c36030bf8963cfa#e6c998715ada4ba6ab40633675124a8a)

@masvelio pls chk w @jnaviask before starting this in progress

**Comment by masvelio** [comment]
@CowMuon [as agreed with Jake](https://hicommonwealth.slack.com/archives/C04D82MDZ3N/p1673365063647329), we should postpone it till the react app will be more functional, as it is not a blocker for the transition

**Comment by jnaviask** [comment]
Wont fix for now.
